### DISCUSSION TITLE
Rarity Function

### DISCUSSION THREAD

**Discussion by jjclay** [nftchef/art-engine#146] [Help + Q&A]
Hello,
In order to get the rarity.js function to work, I had to comment out 2 references to csv:

// const createCsvWriter = require("csv-writer").createObjectCsvWriter;
// outputRarityCSV(ordered);

It seems that csv-writer is missing from the environment.  Is there something that I can do to be able to export my rarity data to a csv file?

Thank you!

**Comment by nftchef** [comment]
you will need to `yarn add csv-writer` or `npm install --save csv-writer`

**Reply by jjclay** [reply]
Thank you!  I'll give that a try.

**Comment by Roskaiu** [comment]
I'm stuck here is the last thing I need to get the project ready. 

You were able to fix it.
